package.json

```diff
@@ -10,6 +10,7 @@
   "main": "./index.js",
   "browser": "./standalone.js",
   "unpkg": "./standalone.js",
+  "module": "./standalone.mjs",
```

Comment:
Having both `standalone.js` and `standalone.mjs` at the same location may cause problem when import without extensions, maybe we can put it in `esm/standalone.mjs`?